Configuration/src/main/java/io/deephaven/configuration/Configuration.java

```diff
@@ -98,69 +105,83 @@ public static Configuration getNamed(@NotNull final String name) throws Configur
-        }
-
-        // The quiet property is available because things like shell scripts may be parsing our System.out and they
-        // don't want to have to deal with these log messages
-        if (!isQuiet()) {
-            log.info().append("Configuration: configuration file is ").append(configurationFile).endl();
-        }
-    }
-
     /**
-     * Clear all currently loaded configurations so that they may be loaded anew.
+     * Clear all currently loaded Configurations so that they may be loaded anew.
      */
     public static void reset() {
         DEFAULT = null;
         NAMED_CONFIGURATIONS.clear();
     }
 
     /**
-     * Clear the specified named configuration so it may be loaded anew.
-     * 
-     * @param name the configuration to clear.
+     * Clear the specified named Configuration so it may be loaded anew.
+     *
+     * @param name the Configuration to clear.
      */
-    public static void reset(final @NotNull String name) {
+    public static void reset(@NotNull final String name) {
         NAMED_CONFIGURATIONS.remove(name);
     }
 
-    public static Configuration newConfigurationForTesting() {
-        return newConfigurationForTesting(DEFAULT_CONF_NAME, DefaultConfigurationContext::new);
+    /**
+     * Create a new, non cached, default Configuration instance.
+     *
+     * @return a new Configuration instance, guaranteed to not be cached.
+     */
+    public static Configuration newStandaloneConfiguration() {
+        return newStandaloneConfiguration(DEFAULT_CONF_NAME, DefaultConfigurationContext::new);
     }
 
-    public static Configuration newConfigurationForTesting(final @NotNull String name) {
-        return newConfigurationForTesting(name, DefaultConfigurationContext::new);
+    /**
+     * Create a new, non cached, named Configuration instance.
+     *
+     * @param name the configuration name
+     * @return a new Configuration instance, guaranteed to not be cached.
+     */
+    public static Configuration newStandaloneConfiguration(@NotNull final String name) {
+        return newStandaloneConfiguration(name, DefaultConfigurationContext::new);
     }
 
+    /**
+     * Create a new, non cached, named Configuration instance.
+     *
+     * @param name the configuration name
+     * @param contextSupplier the supplier for {@link ConfigurationContext}
+     * @return a new Configuration instance, guaranteed to not be cached.
+     */
     @SuppressWarnings("UnusedReturnValue")
-    public static Configuration newConfigurationForTesting(final @NotNull String name,
-            final @NotNull Supplier<ConfigurationContext> contextSupplier) {
-        final Configuration newConfig = DEFAULT_CONF_NAME.equals(name)
+    public static Configuration newStandaloneConfiguration(@NotNull final String name,
+                                                           @NotNull final Supplier<ConfigurationContext> contextSupplier) {
+        return DEFAULT_CONF_NAME.equals(name)
```

Comment:
Instead of making the default name special and then having to look for it here, why not just return a default from the no-arg method? You can offer one more overload that takes a context supplier and no name. Maybe even have this overload reject the name arg if it matches the default, since arguably we want that name to be an internal "secret". This is aligned with another comment I made, to just delete the "default" name constant and completely separate default and named.